### ISSUE TITLE
ff_warpath rework

### ISSUE THREAD

**Issue by fpsmoto** [fortressforever/fortressforever#220]
There are a few issues with ff_warpath that I think could easily be addressed in the lua file/includes files or few edits to the map file. This is a decent map but most of the time it gets played, it becomes too one sided after one team captures 5 points, making it very difficult to come back from if you are on the losing team.
- Replace dettable wall with dettable security lasers so friendly team can access this tunnel as another way to get to CP3, but the enemy team cannot unless detted. (maybe put a generator inside the tunnel near the tunnel exit that could be disabled/repaired similar to ff_dropdown?)
- Lower the amount of points earned for holding capture points. Currently, you get 1 point holding 1 CP, 3 points for holding 2 CPs, 6 for 3 CPs, 10 for 4 and 15 for 5 CPs, plus 25 for capping all 5 points.

It should give 1 for 1, 2 for 2, 3 for 3, 4 for 4 and 5 for 5, plus 25 point bonus for all.

Also, the method in which points are awarded is kind of weird. Say I cap 3 points. Instead of giving me 5 points every X amount of seconds, it gives 1 for CP1, 2 for CP2 and 3 for CP3 all separately. So if the score is 20, it should calculate the total amount of CPs you have and use a value based on that, rather than separately. It's a bit difficult to explain, so here's a video of how it works: https://www.youtube.com/watch?v=JFGiJHMunzI&hd=1
(pay close attention to the Blue team score)

**Comment by FDA-** [comment]
I think the main reason warpath doesn't get played is because it's poopy. I think it's pretty easy with larger player numbers for the map to come into gridlock between midpoint and point 2. I think this is because the points are too far apart, the main path is a very long choke (where it might be better as two chokes with more breathing room between them), and the alternate paths are just as long, being somewhat out of the way as well as some what of a hassle for some classes.

Also I think scoring is retarded. If you cap all the points you win the round, win the most rounds and you win the game. If the map is an eternal stalemate, then you have shamed your team and you should uninstall.

**Comment by AfterShockFF** [comment]
I agree that the map becomes gridlocked too easily. I think it would benefit from stronger push/pull mechanics so that capturing a point really gives your team a chance to push forwards. One thing I've noticed is that if you capture a 4th point, it's only locked for 10 seconds which is barely enough time for the moved respawn to affect the game, and if defenders capture it back, it's then locked for 20 seconds. I think raising the lock time to 15 or 20 there for offense would make it more meaningful. 

That said, I don't particularly like the area between point 4 and 5. That one small corridor is a real spam fest so i'd probably vote to open that up a bit more somehow if we were to encourage more back/forth. Also point 5 is a little tough to defend once people are inside.

I also think the map could benefit from a bit more colour.

I haven't fully thought through the scoring proposals but they sound OK.

**Comment by fpsmoto** [comment]
That's a good idea shock, if you get killed near CP3 right after the enemy team caps CP2, it becomes difficult to take back that CP, though I think the lock times for each cap should be the same for both capture/recapture.

It should also be more difficult to capture a CP the more you've captured. I rarely see CP1 or CP5 get any action because it's currently the opposite. CP1/5 takes very short time to capture and leaves no time to setup a defense in that area. The lock times being adjusted may help with this, but ultimately the amount of time it takes to cap needs to be increased. I remember back in my TFC days having huge battles at each CP including 1 and 5. 

That tunnel near CP2/4 spawn that leads to the upper area near the CPs is pretty much useless except for using the Sniper class or if you know how to trimp/conc well. It's out of the way and cannot be seen as a viable option to get past the choke point near CP2/4 spawn.

+1 for an easter egg that allows you to get above the skybox like in TFC and lay detpacks from the sky above CP3 lol. kidding, ofc.
